### ISSUE TITLE
Knobby's Belt

### ISSUE THREAD

**Issue by Redisarcastic** [slowtorta/turtlewow-bug-tracker#6114]
**Please, provide a clear description what the bug is:**
All the mobs in the cave "evade" and "absorb" but still hit me (imp minions specifically)

**Steps to reproduce the behavior with as much detail as possible:**

1. I walked in the cave. First few mobs outside were fine.
2. Once I got in the cave, it happened with every mob I encountered including Knobby
3. I left because they all started respawning and I'm hardcore. I'm also a warlock, I had my imp out. Idk if it would be class specific.

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

NPC: https://database.turtle-wow.org/?npc= Knobby, Burning Blade mobs inside the cave
Item: https://database.turtle-wow.org/?item=
Quest: https://database.turtle-wow.org/?quest= Knobby's Belt
Spell: https://database.turtle-wow.org/?spell=
Object: https://database.turtle-wow.org/?object=

**Expected behavior. Describe how it should work:**
Not evading me and acting normal lol sorry Idk anything about any technical stuff

**Comment by GrizB37** [comment]
What happens is people fly in with the temporary Wind Rider mount from Razor Hill, and aggro all the mobs on it, while flying straight to Knobby at the back of the cave, this then bugs the mobs out.

**Comment by twowTurtReynolds** [comment]
I dont think that is responsible. On the PTR this is reproducable immediately and nobody's using those custom flying wind riders here.
![image](https://github.com/slowtorta/turtlewow-bug-tracker/assets/151180101/a18bfa65-0370-4d62-baf6-b10b66579faa)
